### ISSUE TITLE
apktool does not work properly

### ISSUE THREAD

**Issue by songmw90** [msintuneappsdk/intune-app-wrapping-tool-android#67]
While testing to build another local environment, I found a problem that did not work in the new environment (my local environment, which is used in the past, wraps without problems)
[success.log](https://github.com/msintuneappsdk/intune-app-wrapping-tool-android/files/8867455/success.log)
[intune_fail.log](https://github.com/msintuneappsdk/intune-app-wrapping-tool-android/files/8867458/intune_fail.log)

I have attached two logs.

As you can see from the log, it seems that decompilation in apktool fails. However, if you download and run the apktool separately, the app will decompile normally.

Do you know anything about this issue?
apk is the same file, only the computer is different

**Comment by bannus** [comment]
Hi @songmw90, I didn't see any failure messages intune_fail.log -- the last log message was "Loading resource table...". Is there anything missing, or is this what you're seeing locally?

I also noticed that the logs are for two different APKs (DSForMobile_v2.5.123_SOFTCAMP_test_20220609.apk vs DSForMobile_v25-release.apk). Can you confirm that it does reproduce for the same APK?

**Comment by songmw90** [comment]
hi @bannus, 

I am attaching the log of attempts with the same apk. As you can see, the successful log is long, and the unsuccessful one is not even called a failure. That's all the log

Literally, it "stops" like that.
I'm calling it a failure for convenience, but it's exactly 'stopped'.

[Fail_LoadingResourceTable.log](https://github.com/msintuneappsdk/intune-app-wrapping-tool-android/files/9139439/Fail_LoadingResourceTable.log)
[success.log](https://github.com/msintuneappsdk/intune-app-wrapping-tool-android/files/9139442/success.log)

**Comment by meghandaly** [comment]
@songmw90 You mentioned "apk is the same file, only the computer is different". 

Can you confirm that both computers satisfy the pre-condition for running the app wrapper that are found in [Fulfill the prerequisites for using the App Wrapping Tool](https://docs.microsoft.com/en-us/mem/intune/developer/app-wrapper-prepare-android#fulfill-the-prerequisites-for-using-the-app-wrapping-tool).

Based on your observations it appears most likely to be an issue with your new environment.

**Comment by rygo-msft** [comment]
@songmw90 I believe that you are likely running the App Wrapping Tool with an incompatible Java version. Can you please run the tool with the `-Verbose` flag. This will provide additional details about the runtime environment.

**Comment by songmw90** [comment]
hi @meghandaly , @rygo-msft 

I've been a bit busy and forgot about it. Thank you for not forgetting to reply.
After reading your comments, I checked the environment again, and there was a difference.
The successful Java version was openjdk 15.0.1, and the openjdk version set on the new computer was 16.0.2.
I saw the comments and set it to 15.0.1, and the wrapping was successful.

I am glad that it succeeded, but I am a little puzzled that it failed in the latest version of Java.

Anyway, we will use a lower version. Thank you for your hard work.
